test/components/viral/alert_component_test.rb

```diff
@@ -15,5 +15,11 @@ class AlertComponentTest < ViewComponentTestCase
       assert_text 'This is an alert alert'
       assert_selector 'div.text-red-800.border-red-300.bg-red-50', count: 1
     end
+
+    test 'alert with content' do
+      render_inline(Viral::AlertComponent.new(message: 'This is an alert alert',
```

Comment:
**suggestion:** When testing components that have content, it is way easier to do and read to use the  `render_preview` as done in some of the other tests.  This allows for full HTML markup.

Reply:
Oh ok. It also makes me create a preview. Please see 68ed92cb79eb7d7a263ee606f6bd8713ec1ba257.

Reply:
I left you a small comment on the commit about this.  Just add back the assert\_text